turbiniactl

```diff
@@ -211,6 +211,12 @@ if __name__ == '__main__':
       '--dump_json',
       action='store_true',
       help='Dump JSON output of Turbinia Request instead of sending it')
+  parser.add_argument(
+      '-p',
+      '--poll_interval',
+      default=60,
+      type=int,
+      help='Number of seconds to wait between poll cycles')
```

Comment:
poll of what? Would be nice to say what is being polled here

Reply:
Thanks -- Fixed this up a bit.